test/runtests.jl

```diff
@@ -9,6 +9,9 @@ using DelimitedFiles
 using Base64
 using Random
 using Statistics: mean
+using StableRNGs
+
+const RNG = StableRNG(1234)
```

Comment:
Does this `RNG` get re-seeded for every test or testset? Otherwise the order of the tests that use `RNG` might affect the outcome of the tests. maybe it is preferable to use the constructor explicetly in ever test.

Reply:
Yes, probably using a new rng is better in the tests instead of re-seeding the rng at the beginning of each `test`, will fix.

Reply:
Fixed.